app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpFragment.kt

```diff
@@ -7,10 +7,13 @@ import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import org.jellyfin.androidtv.data.service.BackgroundService
 import org.jellyfin.androidtv.databinding.FragmentNextUpBinding
+import org.jellyfin.androidtv.preference.UserPreferences
 import org.koin.android.ext.android.inject
 import org.koin.androidx.viewmodel.ext.android.sharedViewModel
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.get
 
-class NextUpFragment : Fragment() {
+class NextUpFragment : Fragment(), KoinComponent {
```

Comment:
```suggestion
class NextUpFragment : Fragment(){
```

AFAIK koin is already available in fragments